projects/plugins/crm/includes/ZeroBSCRM.Core.Localisation.php

```diff
@@ -319,10 +319,11 @@ function zeroBSCRM_date_forceEN( $time = -1 ) {
 		// dd MMMM yyyy HH:mm:ss (IntlDateFormatter - locale based date)
 		// (https://www.php.net/manual/en/class.intldateformatter.php)
 
-		$fmt = new IntlDateFormatter( 'en_US', IntlDateFormatter::FULL, IntlDateFormatter::FULL );
-		$fmt->setPattern( 'dd MMMM yyyy HH:mm:ss' );
-		$r = $fmt->format( $time );
-
+		// phpcs:disable
```

Comment:
Better to ignore only the rules necessary instead of disabling everything.

Reply:
That's fair enough, they were acting as a reminder of sorts but that would be better as a comment. Took the opportunity to clean things up a bit and add the todos.